setup/install.py

```diff
@@ -12,4 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""A setup_forseti wizard for assistance in installation."""
+""" Forseti Installer.
```

Comment:
A couple of comments:
1: the name of this module should be a noun, i.e. instead of `install.py`, it should be `installer.py`
2: please add a description that this is for installing to GCP

Reply:
done